cluster/cluster.yaml

```diff
@@ -45,7 +45,7 @@ Resources:
         Enabled: true
         Timeout: 60
       ConnectionSettings:
-        IdleTimeout: 300
+        IdleTimeout: 4000
```

Comment:
maybe 3600 would be a better time, because it maps to 1h and would remind us better

Reply:
Good point, and the difference is not that big anyway.